tools/c7n_azure/c7n_azure/output.py

```diff
@@ -89,6 +91,7 @@ def join(*parts):
         return "/".join([s.strip('/') for s in parts if s != ''])
 
     @staticmethod
-    def get_blob_client_wrapper(output_path):
+    def get_blob_client_wrapper(output_path, ctx):
```

Comment:
I am assuming it was easier to test, hence we made this a static method. Otherwise, we can pull the context off of `self`

Reply:
Keeps it easier to do `AzureStorageOutput.get_blob_client_wrapper = gm = mock.MagicMock()`